scripts/git-code-review.rb

```diff
@@ -203,7 +203,14 @@ def scan_diffs(cfg)
       pp rev.reviewers
 
       msg  = "*#{rev.author_name}* "
-      msg += "<#{cfg.diff_url}#{rev.revision.id}|#{rev.revision.fields['title'].gsub(/[<>"'\\\/]/, '')}> "
+      
+      title = rev.revision.fields['title'].gsub(/[<>"'\\\/]/, '')
+      # Don't include diff that have WIP in the diff title message
+      if title.downcase.include? "(wip)"
```

Comment:
use `.downcase =~ /[\[\)]wip[\]\)]/`?

Reply:
need to do a `.compact`  to line 223. maybe the title check can be done in the above revs map loop first line

Reply:
Updated. Not sure about the `.compact` though